daemon/config/config_windows.go

```diff
@@ -51,6 +52,11 @@ func (conf *Config) GetInitPath() string {
 	return ""
 }
 
+// LookupInitPath returns an absolute path to docker-init (unimplemented on Windows)
+func (conf *Config) LookupInitPath() (string, error) {
+	return "", errors.New("docker-init is unimplemented on Windows")
+}
+
```

Comment:
`(*Config).LookupInitPath` is only referenced from build-constrained files so this stub should not be necessary.

Reply:
Oh, neat!  I will update. :bow: